readthedocs/projects/forms.py

```diff
@@ -759,12 +759,16 @@ class EnvironmentVariableForm(forms.ModelForm):
 
     class Meta:
         model = EnvironmentVariable
-        fields = ('name', 'value', 'project')
+        fields = ('name', 'value', 'public', 'project')
 
     def __init__(self, *args, **kwargs):
         self.project = kwargs.pop('project', None)
         super().__init__(*args, **kwargs)
 
+        # Remove the nullable option from the form
+        self.fields['public'].widget = forms.CheckboxInput()
+        self.fields['public'].empty_value = False
```

Comment:
Shouldn't we write a data migration that set `public=False` to all the existing ones instead?

Reply:
yeah, but we need to do the migration in the next deploy so we don't have downtime. There are a lot of these missing a migration, may add a PR migrating all of them